source/packages/integration-tests/src/step_definitions/device-patcher/deployment.step.ts

```diff
@@ -295,6 +294,7 @@ Then(
                 patchTask,
                 getAdditionalHeaders(world.authToken)
             );
+            // console.log('patchTaskId: ' + this['patchTaskId']);
```

Comment:
Don't we need to remove this?

Reply:
I kept these because it's annoying to re-add when you're debugging, but I didn't want them there by default. They're in the tests, so I don't think they hurt anything. Can remove if you feel strongly about it.

Reply:
If we want them while debugging, would it make more sense to uncomment them and use console.debug?

Reply:
I changed it to use the logger (I'm not sure what's setting the console.log log level, but it's not working as expected). This consolidates in on the logger anyway, and we set that log level in the integ test script.